src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
@wkurniawan07, I still dont get the difference you are talking about. Over here you have manually done the html verification process as opposed to calling ```verifyHtml```. So unless we test after each of the individual steps in the process, this test adds no value (other than the fact that it would be run more often). I think the test at line 97 in GodModeTest.java accomplishes what this test is doing.

Unless you are saying that you are testing the "process" of html verification by checking whether the steps actually work. In that case, it is not really needed as a call to verifyHtml would do exactly that.